src/App.js

```diff
@@ -19,7 +18,7 @@ function App() {
         <Route path="/login" component={Login} />
         <Route path="/register" component={Register} />
         <Route path="/create" component={CreateHack} />
-        <Route path="/hacks" component={HacksPage} />
+        {/*<Route path="/hacks" component={HacksPage} />*/}
```

Comment:
why are we removing this page?

Reply:
commenting out temporary because in src/components/HacksPage.js line line 68 is being fixed by Joseph in another PR.

Reply:
We should be trying to keep our PRs to just the changes for that feature. depending on merge order of the PRs, we are going to end up with unpredictable results in master.